plugins/MauticFullContactBundle/Sync/Mapping/FieldMappings/CompanyFieldMapping.yaml

```diff
@@ -0,0 +1,61 @@
+---
+-
+    displayName: "Name"
+    name: name
+    type: text
+-
+    displayName: "Location"
+    name: location
+    type: text
+-
+    displayName: "Website"
+    name: website
+    type: text
+-
+    displayName: "LinkedIn"
+    name: linkedin
+    type: text
+-
+    displayName: "Email"
+    name: emails
+    type: text
+-
```

Comment:
Should add required: true

Reply:
This file is just to provide the mapping of source data field and mapping field.
Do we need to parse this attribute and add it on the configuration modal?

Reply:
You have this attribute in Field.php DTO. 
My suggestion make email/company name required and in the first place.

https://github.com/mautic/mautic/pull/8783/files#diff-b095ba0cad61c1b1c2ad02108d685e1cR22

Reply:
Still small improvement.
Maybe email and company email as required.